app/src/main/java/testConfig/Config.kt

```diff
@@ -15,13 +15,15 @@ class Config {
         const val EMAIL: String = "qasdf@gmail.com"
 
         //Existing User
-        const val EXISTING_USER: String ="1marc"
+        const val EXISTING_USER: String ="dfcxc"
+        const val EXISTING_USER2: String ="govind.dixit"
 
         //Non existing User
         const val NON_EXISTING_USER: String = "**33##&&"
 
         //Existing Channel
         const val EXISTING_CHANNEL: String ="general"
+        const val SANDBOX: String ="sandbox"
```

Comment:
@GOVINDDIXIT this seems problematic to me because the sandbox channel is specific to RC's server configuration.  Can these "Existing Channels" be more generic, and then it's up to the user to define the value?  

EXISTING_CHANNEL_1
EXISTING_CHANNEL_2

Something like that?

Also, can you unify this whole file to use UPPER_CASE please?

Reply:
I have updated the PR, Now other forks can replace RC's channel with their one's. Also For UPPER_CASE changes, I have already started working on them on the test branch since It was already requested there, It will be fixed by PR #16